x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_actions_column.tsx

```diff
@@ -39,6 +41,15 @@ export const getUseActionColumnHook =
       ACTION_BUTTON_COUNT--;
     }
 
+    // we do not show the analyzer graph and session view icons on the cases alerts tab alerts table
+    // if the visualization in flyout advanced settings is disabled because these aren't supported inside the table
+    const [visualizationInFlyoutEnabled] = useUiSetting$<boolean>(
+      ENABLE_VISUALIZATIONS_IN_FLYOUT_SETTING
+    );
+    if (!visualizationInFlyoutEnabled && tableId === TableId.alertsOnCasePage) {
+      ACTION_BUTTON_COUNT -= 2;
+    }
```

Comment:
@PhilippeOberti When license in not enterprise and advanced setting is off, the 3 dots still overlaps with the next column. Other scenarios LGTM!
![image](https://github.com/user-attachments/assets/3fe46e2f-b676-4e7a-9f28-73e54549c141)

Reply:
i see now, the session view count was done on line 33, so if it is not enterprise, line 50 should be `-=1`

Reply:
haaaaa good point. Let me fix that! Too many scenarios, I'm glad you're testing it in depth like that 😆

Reply:
I reworked a bit that code and added comments to make it more clear https://github.com/elastic/kibana/pull/195693/commits/b4f1d3da505862f8fc056330930d16ccbd6368a1